scripts/hauler_manifest.yaml

```diff
@@ -0,0 +1,70 @@
+# This is a base file to be used to import all the required resources to run
+# Kubewarden in airgap environment into Hauler.
+#
+# To sync this file with Hauler and load everything into its storage, you can
```

Comment:
I have a question, does this sync include new images automatically, if for example we add a new component or policy?

Reply:
Sorry, I was with the sync/update of the manifest with the latest version (updatecli script) in mind and missed the context of this comment. 

The Hauler sync command will download all the resources defined in the manifest. Therefore, if you add more policies in the manifest, it should download them.